src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapBad.java

```diff
@@ -1,11 +1,13 @@
 package com.google.checkstyle.test. //warn
               chapter3filestructure.rule332nolinewrap;
 
-import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
+import static java.math. //warn
+        BigInteger.ZERO;
 
-import javax.accessibility. //warn
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater; //ok
+import javax.accessibility. //warn
+        AccessibleAttributeSequence;
```

Comment:
this is "Input" file , code in such files are suppose to be ugly as it is possible. Style guides are not applicable to them at all.
shifting of line numbers is ok, if shifting is affect a lot of code it is recommended to create new Input  file and put most new cases in it.